tests/testthat/test_civis_ml.R

```diff
@@ -362,7 +362,7 @@ test_that("calls scripts_post_custom", {
   )
 
   script_args <- mock_args(fake_scripts_post_custom)[[1]]
-  expect_equal(script_args$from_template_id, 1111)
+  expect_equal(script_args$from_template_id, fake_model$job$fromTemplateId + 1)
```

Comment:
This makes me a little nervous because as far as I can tell, it assumes that the prediction template will always be 1 greater than the training template. This is usually the case (we try to do that), but it's not necessarily true, so if you add new templates in a future release, it would break your tests in a surprising way. Can you grab the id from the data frame directly?

Reply:
I'm just going to test `get_predict_template_id` once, and just use it everywhere else.